pkg/cloudprovider/provider/azure/provider.go

```diff
@@ -46,6 +49,10 @@ import (
 )
 
 const (
+	CapabilityPremiumIO = "PremiumIO"
```

Comment:
I'm not sure that makes sense because `PremiumIO` and `UltraSSDAvailable` are not values for the same field in the Azure response necessarily. Having an enum would suggest they are, no?